src/main/java/org/darisadesigns/polyglotlina/ManagersCollections/ImageCollection.java

```diff
@@ -85,34 +60,24 @@ public ImageNode openNewImage(Window parent, File workingDirectory) throws Excep
     public void insert(Integer _id) throws Exception {
         super.insert(_id, bufferNode);
 
-        bufferNode = new ImageNode();
+        bufferNode = new ImageNode(core);
     }
     
-    /**
-     * Takes a buffered image, and returns a node containing it, having inserted
-     * the node with ID to persist on save
-     * @param _image Image to get node of.
-     * @return populated Image node
-     * @throws java.lang.Exception
-     */
-    public ImageNode getFromBufferedImage(BufferedImage _image) throws Exception {
-        ImageNode ret = new ImageNode();
-        ret.setImage(_image);
-        this.insert(ret);
-        
-        return ret;
+    @Override
+    public Integer insert(ImageNode _buffer) throws Exception {
+        return super.insert(_buffer);
     }
 
     @Override
     public ImageNode notFoundNode() {
-        ImageNode emptyImage = new ImageNode();
+        ImageNode emptyImage = new ImageNode(core);
         
         try {
-            emptyImage.setImage(ImageIO.read(getClass().getResource(PGTUtil.NOT_FOUND_IMAGE)));
+            emptyImage.setImageBytes(core.getOSHandler().getIOHandler().loadImageBytes(PGTUtil.NOT_FOUND_IMAGE));
```

Comment:
I do not believe this change will work, even if it compiles. The reference is to a path found within PolyGlot itself, which is why I'm calling class.getResource() in the original code.